runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -89,4 +89,15 @@ default int getPrayerThreshold()
 	{
 		return 0;
 	}
+
+	@ConfigItem(
+			keyName = "customMessage",
+			name = "Input Custom Message",
+			description = "Input the precise message that you want to get a notification for.  Leave empty to disable.",
+			position = 6
+	)
+	default String customMessage()
```

Comment:
I would add list of messages, also wildcard support (check how the npc highlighting, and ground items work)